docs/source/guide/start.md

```diff
@@ -38,19 +141,31 @@ There are several possible ways to run Label Studio with an external domain name
 - Specify the parameters when you start Label Studio: `label-studio start --host http://your.domain.com/ls-root`.
 - Specify the parameters as environment variables `HOST` especially when setting up Docker: `HOST=https://your.domain.com:7777`. 
 
-> Don't forget to specify protocol: `http://` or `https://`
+Or, you can use environment variables:
+```
+LABEL_STUDIO_HOST = https://subdomain.example.com:7777
+```
+
+You must specify the protocol for the domain name: `http://` or `https://`
 
-> If your external host has a port, e.g.: `http://77.77.77.77:1234` then you have to specify HOST with the port together `HOST=http://77.77.77.77:1234`. 
+If your external host has a port, specify the port as part of the host name. 
 
 ## Set up task sampling for your project 
 
-When you start Label Studio, you can define the way of how your imported tasks are exposed to annotators by setting up task sampling. To enable task sampling, specify one of the sampling option with the `--sampling=<option>` command line argument when you start Label Studio. 
+When you start Label Studio, you can control the order in which tasks are exposed to annotators for a specific project.  
+
+For example, to create a project with sequential task ordering for annotators:
+```bash
+label-studio start <project_name> --sampling sequential
+```
 
 The following table lists the available sampling options: 
 
 | Option | Description |
 | --- | --- | 
 | sequential | Default. Tasks are shown to annotators in ascending order by the `id` field. |
 | uniform | Tasks are sampled with equal probabilities. |
-| prediction-score-min | Tasks with the minimum average prediction score are shown to annotators. To use this option, you must also include predictions data in the task data that you import into Label Studio. |
```

Comment:
we've decided to revert `prediction-score-min`

Reply:
so it's back in?

Reply:
yes

Reply:
perf, added it back